tests/ImageSharp.Tests/Processing/Transforms/PadTest.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace SixLabors.ImageSharp.Tests.Processing.Transforms
 {
+    [Trait("Category", "Processors")]
```

Comment:
Might be an idea in the future to create dedicated attributes for these kind of repetitive values.